src/system/SystemPacketBuffer.cpp

```diff
@@ -536,5 +536,31 @@ PacketBufferHandle PacketBufferHandle::PopHead()
     return PacketBufferHandle(head);
 }
 
+PacketBufferHandle PacketBufferHandle::CloneData()
+{
+    if (!mBuffer->Next().IsNull())
```

Comment:
Could this be an assertion?

Reply:
I am unsure how assertions work. nlASSERT?

Reply:
All asserts in this file seem to be static. nlASSERT seem to be a noop in production (really? that seems odd, but this is what following #define seems to give me).

I would rather keep it as is as empty return.